modules/log.cpp

```diff
@@ -268,15 +269,35 @@ CString CLogMod::GetServer()
 
 bool CLogMod::OnLoad(const CString& sArgs, CString& sMessage)
 {
+	CString sMyArgs = sArgs;
```

Comment:
Not sure if this copying is necessary, it was done this way in awaystore.cpp so I kept it like that.